src/applications/mhv-landing-page/containers/LandingPageContainer.jsx

```diff
@@ -67,6 +75,29 @@ const LandingPageContainer = () => {
     [loading],
   );
 
+  useEffect(
+    () => {
+      if (!profile.loading) {
+        if (userHasMhvAccount) {
+          dispatch({
+            type: fetchAccountStatusSuccess,
+            data: { error: false },
+          });
+        } else {
+          dispatch({ type: fetchAccountStatus });
+
+          getMHVAccount().then(resp => {
+            dispatch({
+              type: fetchAccountStatusSuccess,
+              data: resp,
```

Comment:
The mock success & error account fetch are working. But I notice that when testing with account success mock `'GET /v0/user/mhv_user_account': MHVAccountStatus.accountSuccess,`, the network tab shows that we're fetching the `mhv_user_account` 3 times. Not sure if that's something we want to follow up with?

<img width="1197" alt="Screenshot 2024-12-18 at 4 08 26 PM" src="https://github.com/user-attachments/assets/ed4e8399-ab3e-4208-a09f-73dcd35b42ce" />

Reply:
I'm having trouble reproducing this issue locally but it should definitely be addressed.

Reply:
Found what caused the repeated account api fetch.
Changing the dependency array to `[userHasMhvAccount, profile.loading, dispatch],` fixed the issue